bom/pom.xml

```diff
@@ -35,6 +35,7 @@
     <californium.version>2.6.2</californium.version>
     <commons-cli.version>1.4</commons-cli.version>
     <dispatch-router.image.name>quay.io/interconnectedcloud/qdrouterd:1.14.0</dispatch-router.image.name>
+    <flyway-core.version>7.7.2</flyway-core.version>
```

Comment:
It looks like using this specific version would require an Eclipse Contribution Questionnaire (CQ) entry.
There is already an approved entry for version 7.3, so, if the newest version isn't explicitly needed, we could use `7.3.2` here for now (version is from December 2020).

A dependency license check can be done like this:
- run `create_dependencies_list.sh`; this will update `legal/src/main/resources/legal/hono-maven.deps`
- run the [Dash License Tool](https://github.com/eclipse/dash-licenses/): `java -jar <path>/org.eclipse.dash.licenses-0.0.1-SNAPSHOT.jar -summary legal/src/main/resources/legal/DEPENDENCIES legal/src/main/resources/legal/hono-maven.deps`

Reply:
Thank you @calohmn! I will check this.

Reply:
You can do a rebase here and run `create_dependencies_list.sh` with the Dash License Tool jar path as parameter (see #2595).

Reply:
I changed the Flyway version to `7.3.2`.